### PR TITLE
[KYUUBI #1780] Log error message on REST endpoints 

### LINKED ISSUE #1780: [FEATURE] Log error message on REST endpoints
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the feature

Add error log in Rest be

### Motivation

_No response_

### Describe the solution

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by simon824**
### _Why are the changes needed?_
closes #1780 

Add error log in Rest be 

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala [hunk 1]
```diff
@@ -29,6 +29,7 @@ import io.swagger.v3.oas.annotations.tags.Tag
 import org.apache.hive.service.rpc.thrift._
 
 import org.apache.kyuubi.KyuubiSQLException
+import org.apache.kyuubi.Utils.error
```
Comment (yaooqinn) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala:32]: please use Logging trait

Comment (simon824) [reply kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala]: Updated, thx.

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala [hunk 2]
```diff
@@ -53,8 +54,10 @@ private[v1] class OperationsResource extends ApiRequestContext {
       val operation = fe.be.sessionManager.operationManager.getOperation(opHandle)
       KyuubiOperationEvent(operation.asInstanceOf[KyuubiOperation])
     } catch {
-      case NonFatal(_) =>
-        throw new NotFoundException(s"Error getting an operation event")
+      case NonFatal(e) =>
+        val errorMsg = s"Error getting an operation event"
```
Comment (ulysses-you) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala:58]: nit: no `s` before `""`

Comment (simon824) [reply kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala]: Updated, thx.

**Review by yaooqinn — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**